daemon/debuginfo.go

```diff
@@ -42,7 +42,7 @@ func (h *getDebugInfo) Handle(params restapi.GetDebuginfoParams) middleware.Resp
 	dr := models.DebugInfo{}
 	d := h.daemon
 
-	dr.CiliumVersion = fmt.Sprintf("%s (%s) %s", cver.Version, cver.Revision, cver.Arch)
+	dr.CiliumVersion = fmt.Sprintf("%s %s %s", cver.Version, cver.Revision, cver.Arch)
```

Comment:
My current output looks like this

```
$ cilium version
Client: {"Version":"0.13.90","Revision":"377eaf9","GoRuntimeVersion":"go1.9","Arch":"linux/amd64"}
Daemon: {"Version":"0.13.90","Revision":"377eaf9","GoRuntimeVersion":"go1.9","Arch":"linux/amd64"}
```
without the change the server sends back braces arround the Revision 
```
$ cilium version
Client: {"Version":"0.13.90","Revision":"377eaf9","GoRuntimeVersion":"go1.9","Arch":"linux/amd64"}
Daemon: {"Version":"0.13.90","Revision":"(377eaf9)","GoRuntimeVersion":"go1.9","Arch":"linux/amd64"}
```

Reply:
Ah... ok.. thanks!

Reply:
I think this is breaking the jenkins tests. It could be because `dr.CiliumVersion ` is being used at other places, and the format is assumed in a specific way.